tests/php/Authenticator/RegisterHandlerTest.php

```diff
@@ -58,7 +58,7 @@ public function testRegisterRouteIsPrivateWithGETMethod()
      */
     public function testRegisterRouteIsPrivateWithPOSTMethod()
     {
-        $response = $this->post(self::URL, []);
+        $response = $this->post(self::URL, ['dummy' => 'data']);
```

Comment:
Is this payload required? If so, could you comment it to say why?

Reply:
Yeah - https://github.com/silverstripe/silverstripe-framework/pull/8987

I'll add a comment